{"id":28023,"className":"javax.security.sasl.SaslException","methodName":"SaslException","signature":"\u003cjavax.security.sasl.SaslException: void SaslException(java.lang.String,java.lang.Throwable)\u003e","javadoc":"/** \n * Constructs a new instance of \u003ctt\u003eSaslException\u003c/tt\u003e with a detailed message and a root exception. For example, a SaslException might result from a problem with the callback handler, which might throw a NoSuchCallbackException if it does not support the requested callback, or throw an IOException if it had problems obtaining data for the callback. The SaslException\u0027s root exception would be then be the exception thrown by the callback handler.\n * @param detail A possibly null string containing details of the exception.\n * @param ex A possibly null root exception that caused this exception.\n * @see java.lang.Throwable#getMessage\n * @see #getCause\n */\n","tf":{"freq":{"result":1,"might":2,"detail":3,"root":3,"data":1,"possibl":2,"lang":1,"request":1,"sasl":4,"except":12,"thrown":1,"support":1,"messag":2,"problem":2,"instanc":1,"obtain":1,"caus":2,"string":1,"contain":1,"callback":5,"exampl":1,"java":1,"throwabl":1,"construct":1,"handler":2},"maxFreq":12,"totalTerms":53},"loc":6}